capture.py

```diff
@@ -91,6 +91,16 @@ def capture(camera=None,
     """
 
     camera = camera or "persp"
+    
+    # Fallback to default settings
+    if camera_options is None:
```

Comment:
Ah, I see. Because it was already doing that, but this relates to [the next comment](https://github.com/mottosso/maya-capture/pull/28#discussion_r53041607), on removing the context managers? The behaviour went away there.